dist/init/linux-upstart/caddy.conf

```diff
@@ -14,12 +14,12 @@ respawn limit 10 5
 reload signal SIGUSR1
 
 # Let's Encrypt certificates will be written to this directory.
-env CADDYPATH=/etc/caddy
+env CADDYPATH=/etc/ssl/caddy
```

Comment:
So /etc/caddy is actually the HTTP site root? And you need a separate Caddypath which stores OCSP staples and certs... I guess the name 'ssl' is fine. 🤷‍♂️